flink-core/src/main/java/org/apache/flink/api/common/ExecutionConfig.java

```diff
@@ -612,6 +614,42 @@ public boolean hasGenericTypesDisabled() {
 		return disableGenericTypes;
 	}
 
+	/**
+	 * Enables the Flink runtime to auto-generate UID's for operators.
+	 *
+	 * @see #disableAutoGeneratedUIDs()
+	 */
+	public void enableAutoGeneratedUIDs() {
+		enableAutoGeneratedUids = true;
+	}
+
+	/**
+	 * Disables auto-generated UIDs. Forces users to manually specify UIDs
+	 * on their datastreams using {@code DataStream#uid} or {@code DataStream#setUidHash}
```

Comment:
Actually `DataStream` has neither `uid` nor `setUidHash` methods. I think it is enough to say that it forces to use manually specified UIDs.